src/kaltura-player.js

```diff
@@ -518,6 +518,14 @@ class KalturaPlayer extends FakeEventTarget {
     return this._localPlayer.Error;
   }
 
+  get droppedVideoFrames(): ?number {
```

Comment:
need to add them also in [base-remote-player.js](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/cast/base-remote-player.js)